serverless-lambda-container-otel-layer-example/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM public.ecr.aws/lambda/nodejs:12
```

Comment:
Is this a node 12 image? If so, shouldn't we be using node 16 or 18?

Reply:
good catch, will update to 18